packages/itwin/grouping-mapping-widget/src/widget/components/CalculatedPropertyActionWithVisuals.tsx

```diff
@@ -177,7 +280,7 @@ export const CalculatedPropertyActionWithVisuals = ({
               labelPosition='left'
               disabled={isLoading}
               checked={colorProperty}
-              onChange={() => setColorProperty((b) => !b)}
+              onChange={() => setColorProperty((b: any) => !b)}
```

Comment:
Let's avoid `any`.

Reply:
Made changes where ever possible to avoid the use of "any"